### PR TITLE
dwm-status: 1.5.0 -> 1.6.0

### PR DESCRIPTION
**Pull request by Gerschtli**
###### Motivation for this change

Update package.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/window-managers/dwm/dwm-status.nix [hunk 1]
```diff
@@ -1,31 +1,42 @@
 { stdenv, lib, rustPlatform, fetchFromGitHub, dbus, gdk_pixbuf, libnotify, makeWrapper, pkgconfig, xorg
-, enableAlsaUtils ? true, alsaUtils, bash, coreutils }:
+, enableAlsaUtils ? true, alsaUtils, coreutils
+, enableNetwork ? true, dnsutils, iproute, wirelesstools }:
 
 let
-  binPath = stdenv.lib.makeBinPath [
-    alsaUtils bash coreutils
-  ];
+  bins =
```
Comment (timokau) [inline pkgs/applications/window-managers/dwm/dwm-status.nix]:
```suggestion
   bins = lib.optionals enableAlsaUtils [
     alsaUtils
     coreutils
   ] ++ (lib.optionals enableNetwork [
     dnsutils
     iproute
     wirelesstools
   ]);
```

(not sure if the parenthesis are necessary)

File: pkgs/applications/window-managers/dwm/dwm-status.nix [hunk 2]
```diff
@@ -1,31 +1,42 @@
...
   nativeBuildInputs = [ makeWrapper pkgconfig ];
   buildInputs = [ dbus gdk_pixbuf libnotify xorg.libX11 ];
 
-  cargoSha256 = "1cngcacsbzijs55k4kz8fidki3p8jblk3v5s21hjsn4glzjdbkmm";
+  cargoSha256 = "1khknf1bjs80cc2n4jnpilf8cc15crykhhyvvff6q4ay40353gr6";
 
-  postInstall = lib.optionalString enableAlsaUtils ''
-    wrapProgram $out/bin/dwm-status --prefix "PATH" : "${binPath}"
-  '';
+  postInstall =
```
Comment (timokau) [inline pkgs/applications/window-managers/dwm/dwm-status.nix]:
```suggestion
   postInstall = lib.optionalString (bins != []) "...";
```

**Review by timokau — CHANGES_REQUESTED** [review]

File: pkgs/applications/window-managers/dwm/dwm-status.nix (hunk 2 above)
Comment (Gerschtli) [reply pkgs/applications/window-managers/dwm/dwm-status.nix]: Ah thank you! Could not come up with the name, will replace it

File: pkgs/applications/window-managers/dwm/dwm-status.nix (hunk 1 above)
Comment (Gerschtli) [reply pkgs/applications/window-managers/dwm/dwm-status.nix]: I will check it

**Review by timokau — APPROVED** [review]

**Comment by timokau** [comment]
Thanks :)

**Pull request merged**